core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/EntityStatisticsCacheMonitor.scala

```diff
@@ -26,10 +25,14 @@ object EntityStatisticsCacheMonitor {
   sealed trait EntityStatisticsCacheMessage
   case object Sweep extends EntityStatisticsCacheMessage
   case object ScheduleDelayedSweep extends EntityStatisticsCacheMessage
+
+  //Note: Ignored workspaces have a cacheLastUpdated timestamp of 1000ms after epoch
+  val MIN_CACHE_TIME = new Timestamp(1000)
+
 }
 
 class EntityStatisticsCacheMonitorActor(val dataSource: SlickDataSource, val timeoutPerWorkspace: Duration, val standardPollInterval: FiniteDuration, val workspaceCooldown: FiniteDuration)(implicit val executionContext: ExecutionContext) extends Actor with EntityStatisticsCacheMonitor with LazyLogging {
-  import context._
+  import context.setReceiveTimeout
```

Comment:
`import context._` confused IntelliJ and made the `pipeTo` on line 45 appear in red; I've tweaked this line to remove IntelliJ warnings.